<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QDialog" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>669</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>935</width>
    <height>669</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>935</width>
    <height>669</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(50, 50, 50);
}</string>
  </property>
  <widget class="QLabel" name="label_sum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string> Сумма вклада, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_period">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Срок, мес</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_procentRate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Процентная ставка, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_taxRate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Налоговая ставка, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_periodOfPay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Период выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_taxes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: rgb(182, 56, 56);</string>
   </property>
   <property name="text">
    <string>Налог</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sumProcent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: rgb(182, 56, 56);</string>
   </property>
   <property name="text">
    <string>Доход, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_remainSum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: rgb(182, 56, 56);</string>
   </property>
   <property name="text">
    <string>Сумма в конце срока, руб</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25px;
border: 1px solid gray;
color: white;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>9223372036854775808.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_procent">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25px;
border: 1px solid gray;
color: white;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_taxProcent">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_period">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25px;
border: 1px solid gray;
color: white;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_taxesToPay">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(182, 56, 56);;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sumOfProcentEnd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(182, 56, 56);;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sumRemain">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(182, 56, 56);;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_once">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Единоразово</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_eachMonth">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Раз в месяц</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>381</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(75, 75, 75);
	color: white;
	border: 1px solid gray;
	border-radius: 40px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									stop: 0 #dadbde, stop: 1 #D3D3D3);
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addDeposit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>570</y>
     <width>261</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(75, 75, 75);
	color: white;
	border: 1px solid gray;
	border-radius: 40px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									stop: 0 #dadbde, stop: 1 #D3D3D3);
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>570</y>
     <width>241</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(211, 211, 211);
	color: rgb(182, 56, 56);
	border: 1px solid black;
	border-radius: 45px;
	border-style: outset;
}
QPushButton:pressed {
	background-color: rgb(182, 56, 56);
	color: rgb(211, 211, 211);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_add">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>251</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_min">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>251</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_capitalization">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>КАПИТАЛИЗАЦИЯ</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_addition">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>ПОПОЛНЕНИЕ/СНЯТИЕ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
